src/store/create-store.js

```diff
@@ -44,27 +44,29 @@ export default async function createStore(
   if (detectEnv.isBrowser()) {
     try {
       // sync redux state with browser storage
-      const reduxState = await bs.syncReduxState(initialState)
+      const reduxState = await bs.syncReduxState(preloadedState)
       const store = _createStore(rootReducer, reduxState, storeEnhancer)
       if (isDev && module.hot) {
         // Enable Webpack hot module replacement for reducers
+        // Or it will cause multiple store instances to show in Redux Dev Tools.
         module.hot.accept('../reducers', () => {
-          const nextRootReducer = require('../reducers').default
-          store.replaceReducer(nextRootReducer)
+          store.replaceReducer(require('../reducers').default)
         })
       }
       // Subscribe the redux store changes.
       // Sync the browser storage after redux state change.
+      // Use `debounce` (instead of throttle) to save the last result in duration into storage
       store.subscribe(
-        _.throttle(() => {
+        _.debounce(() => {
           bs.syncReduxState(store.getState())
-        }, 1000)
+        }, 3000)
```

Comment:
`debounce` is ok for me as well.
However, [the main difference between throttle and debouncing is that throttle guarantees the execution of the function regularly, at least every X milliseconds](https://css-tricks.com/debouncing-throttling-explained-examples/#article-header-id-5).